From 261180999c7b8a17b22e6a7a2e0e4fb3b90eb849 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 29 Dec 2010 07:09:58 +0800
Subject: [PATCH 02/11] ENGR00137445 Update Android.mk to add
 LOCAL_MODULE_TAGS := optional

Update Android.mk to add LOCAL_MODULE_TAGS := optional
Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 android/Android.mk | 4 ++--
 pppd/Android.mk    | 7 +++++++
 2 files changed, 9 insertions(+), 2 deletions(-)

diff --git a/android/Android.mk b/android/Android.mk
old mode 100644
new mode 100755
index e40bb81..7de14a0
--- a/android/Android.mk
+++ b/android/Android.mk
@@ -21,7 +21,7 @@ LOCAL_SRC_FILES := ip-up-ppp0.c
 LOCAL_SHARED_LIBRARIES := libcutils
 LOCAL_MODULE := ip-up-ppp0
 LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/ppp
-
+LOCAL_MODULE_TAGS := optional
 include $(BUILD_EXECUTABLE)
 
 include $(CLEAR_VARS)
@@ -30,5 +30,5 @@ LOCAL_SRC_FILES := ip-down-ppp0.c
 LOCAL_SHARED_LIBRARIES := libcutils
 LOCAL_MODULE := ip-down-ppp0
 LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/ppp
-
+LOCAL_MODULE_TAGS := optional
 include $(BUILD_EXECUTABLE)
diff --git a/pppd/Android.mk b/pppd/Android.mk
old mode 100644
new mode 100755
index f0825bc..ffba574
--- a/pppd/Android.mk
+++ b/pppd/Android.mk
@@ -43,5 +43,12 @@ LOCAL_CFLAGS := -fno-short-enums -DCHAPMS=1 -DMPPE=1 -DINET6=1 -DUSE_OPENSSL=1 -
 LOCAL_CFLAGS += -Wno-unused-variable
 
 LOCAL_MODULE:= pppd
+LOCAL_MODULE_TAGS := optional
+include $(BUILD_EXECUTABLE)
 
+include $(CLEAR_VARS)
+LOCAL_SRC_FILES := chat.c
+LOCAL_SHARED_LIBRARIES := libcutils
+LOCAL_MODULE := chat
+LOCAL_MODULE_TAGS := optional
 include $(BUILD_EXECUTABLE)
-- 
1.8.0

